core/themes/seven/css/style.css

```
@@ -803,7 +803,13 @@ form.views-exposed-form {
 #autocomplete li {
   padding: 0 5px;
 }
-
+.autocomplete-description {
```

I know that we've discussed reducing this to just the specific class name, but in order to fix the gray in the selected/hovered list items, I had to add the following `#autocomplete li.selected .autocomplete-description` selector, so this now seems kinda odd/silly. Should I change it back to the "full" version?:

```suggestion
#autocomplete li .autocomplete-description {
```